### PR TITLE
Add database/

### PR DESCRIPTION
**Pull request by JaneAarthy**
Adding ER model and database scripts for StratosManager

### CODE REVIEW

**Comment by imesh** [comment]
Thanks @JaneAarthy for the pull request. We might need to wait until we release Stratos 4.1.0 to merge this pull request.

**Comment by imesh** [comment]
@JaneAarthy Shall we move this feature implementation to database-model branch? If so may be you can close this pull request and send a new one.
Thanks

File: components/org.apache.stratos.manager/src/main/java/org/apache/stratos/manager/persistence/PersistenceManager.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+
+import javax.persistence.Persistence;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by aarthy on 7/6/15.
+ */
```
Comment (Vishanth) [inline components/org.apache.stratos.manager/src/main/java/org/apache/stratos/manager/persistence/PersistenceManager.java]: @JaneAarthy we don't put created by comment. We only put the Apache License header for the classes.

**Comment by JaneAarthy** [comment]
moving the changes to the database-model branch

File: components/org.apache.stratos.manager/src/main/database/SM.sql [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+CREATE SCHEMA IF NOT EXISTS `StratosManager` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci ;
+USE `StratosManager` ;
+
+-- -----------------------------------------------------
+-- Table `StratosManager`.`Cluster`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `StratosManager`.`Cluster` (
+  `clusterId` VARCHAR(255) NULL,
```
Comment (sajhak) [inline components/org.apache.stratos.manager/src/main/database/SM.sql]: @JaneAarthy , why this clusterId, which is also declared as the Primary key,  is defined as a 'NULL' ?

**Pull request closed without merging**